hack/genref/markdown/pkg.tpl

```diff
@@ -0,0 +1,46 @@
+{{ define "packages" -}}
+
+{{- range $idx, $val := .packages -}}
+{{/* Special handling for config */}}
+  {{- if .IsMain -}}
+---
+title: {{ .Title }}
+content_type: tool-reference
+package: {{ .DisplayName }}
+auto_generated: true
+description: Generated API reference documentation for {{ if ne .GroupName "" -}} {{ .DisplayName }}{{ else -}} Kueue Configuration{{- end -}}.
```

Comment:
this part `Kueue Configuration` We shouldn't need

Reply:
Updated. PTAL

Reply:
do we have `LeaderWorkerSet Configuration` ? if not, we should remove this part

Reply:
see https://github.com/kubernetes-sigs/lws/issues/170